packages/experience-builder-sdk/src/blocks/preview/CompositionBlock.tsx

```diff
@@ -73,19 +75,31 @@ export const CompositionBlock = ({
   }, [isAssembly, node.definitionId]);
 
   const nodeProps = useMemo(() => {
-    const propMap: Record<string, PrimitiveValue> = {
-      cfSsrClassName: node.variables.cfSsrClassName
-        ? resolveDesignValue(
-            (node.variables.cfSsrClassName as DesignValue).valuesByBreakpoint,
-            'cfSsrClassName',
-          )
-        : undefined,
-    };
     // Don't enrich the assembly wrapper node with props
     if (!componentRegistration || isAssembly) {
-      return propMap;
+      return {
+        cfSsrClassName: node.variables.cfSsrClassName
+          ? resolveDesignValue(
+              (node.variables.cfSsrClassName as DesignValue).valuesByBreakpoint,
+              'cfSsrClassName',
+            )
+          : undefined,
+      };
     }
 
+    const propMap: Record<string, PrimitiveValue> = {
+      // @ts-expect-error -- bla
+      cfSsrClassName: node.id
+        ? // @ts-expect-error -- bla
```

Comment:
I know it's always the same reason but I would really avoid leaving a comment like "bla" in the code.
]
```suggestion
      // @ts-expect-error -- node receives a random ID in ssrStyles (no type support yet)
      cfSsrClassName: node.id
        ? // @ts-expect-error -- node receives a random ID in ssrStyles (no type support yet)
```